Rename 'excludemonofailures' to 'mono'

trylek opened the pull request:
Fixes: https://github.com/dotnet/runtime/issues/62745

/cc @dotnet/runtime-infrastructure

trylek committed:
Rename 'excludemonofailures' to 'mono'

Fixes: https://github.com/dotnet/runtime/issues/62745

jkoritzinsky commented:
Do we want to keep compatibility with the old argument name at least until the next infra rollout so we don't break people when they get back from vacation?

trylek commented:
@jkoritzinsky - I think that's a valid concern, we could keep the two places in build.cmd and build.sh; I defer to Mono folks to chime in how much of a real concern that is.

trylek committed:
Keep the old build.cmd/sh script options per Jeremy's PR feedback

We'll delete them in the next infra rollout, this is just to make
the transition smoother with fewer unpleasant surprises adding
to the New Year hangover.

Thanks

Tomas
